### PR TITLE
Flower order app in Vue

### PR DESCRIPTION
**Pull request by andiwolfpack**

### CODE REVIEW

**Comment by AdrianFlorian** [comment]
can we have a small gif? 👍

File: my-app/src/components/HelloWorld.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<template>
```
Comment (AdrianFlorian) [inline my-app/src/components/HelloWorld.vue]: maybe the name could  be better here?

File: my-app/src/components/andi.vue [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<template>
```
Comment (AdrianFlorian) [inline my-app/src/components/andi.vue]: what about this one?

**Review by AdrianFlorian — CHANGES_REQUESTED** [review]
Is this a list -> detail view or just the list?

File: my-app/public/index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+<head>
+  <link href='https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900|Material+Icons' rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/vuetify/dist/vuetify.min.css" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
+</head>
+
+<body>
+  <div id="app">
+    <v-app>
```
Comment (robikovacs) [inline my-app/public/index.html]: You have a duplicate `v-app` component declared here, and in `App.vue`. `v-app`, `v-content` and `v-container` are not rendered at all, you can delete those. Leave only `<div id="app">`

File: my-app/public/index.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      <v-content>
+        <v-container>Hello world</v-container>
+      </v-content>
+    </v-app>
+  </div> 
+
+  <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/vuetify/dist/vuetify.js"></script>
+  <script>
```
Comment (robikovacs) [inline my-app/public/index.html]: You can remove this block of code. You already use `new Vue` in `main.js`

**Review by robikovacs — CHANGES_REQUESTED** [review]
Good job 👍, just some (common) setup issues left.

File: my-app/src/components/andi.vue (hunk 2 above)
Comment (robikovacs) [reply my-app/src/components/andi.vue]: `scirocco.vue`

File: my-app/public/index.html (hunk 4 above)
Comment (robikovacs) [reply my-app/public/index.html]: Now, a tricky question for you: where is `main.js` imported? Hint: `npm run serve` and `npm run build`.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,3 @@
 # Flower orders in Vue
 Vue simple project to get flower orders
+YES!
```
Comment (florinionce) [inline README.md:3]: YES

File: my-app/src/App.vue [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+    <v-content>
+      <HelloWorld v-show="show"></HelloWorld>
+    </v-content>
+  </v-app>
+</template>
+
+<script>
+  import HelloWorld from './components/HelloWorld'
+  import Andi from './components/andi.vue'
```
Comment (florinionce) [inline my-app/src/App.vue]: :andi:

File: my-app/public/index.html (hunk 4 above)
Comment (florinionce) [reply my-app/public/index.html]: Nice @robikovacs

File: my-app/src/App.vue (hunk 6 above)
Comment (florinionce) [reply my-app/src/App.vue]: too bad it's not used :(

File: my-app/src/components/HelloWorld.vue [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  <div>
+    <v-data-table :headers="headers" :items="orders" hide-actions class="elevation-1">
+      <template slot="items" slot-scope="props">
```
Comment (florinionce) [inline my-app/src/components/HelloWorld.vue]: where do you have the `items` `<slot>` used? 
@andiwolfpack  @robikovacs  correct me if I'm wrong, but I think you need this code somewhere:
```
<items>
   <slot></slot>
</items>
```
and then your items from here will be inserted there. If you don't have this, is there a reason to use slots here?

File: README.md (hunk 5 above)
Comment (robikovacs) [reply README.md:3]: That's my contribution to the project. (proud)

File: my-app/src/components/HelloWorld.vue (hunk 7 above)
Comment (robikovacs) [reply my-app/src/components/HelloWorld.vue]: @florinionce that is the name of the slot, `"items"`, the code is correct. Check "Scope slots" in [here](https://vuetifyjs.com/en/components/data-tables#api)

File: my-app/src/components/HelloWorld.vue [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+          text: 'Deliver to',
+          value: 'deliver_to'
+        }
+      ],
+      orders: [],
+      answer: ''
+    }),
+    created: function () {
+      let _this = this
+      axios.get('http://demo5252393.mockable.io/flowers')
```
Comment (robikovacs) [inline my-app/src/components/HelloWorld.vue]: You can declare constants, like the url, using `const API_URL = "http..."`. The best place to declare them is between the `import` and `export default`.

File: my-app/src/components/HelloWorld.vue (hunk 7 above)
Comment (florinionce) [reply my-app/src/components/HelloWorld.vue]: Oh, so the slot is defined by vuetify. Thanks :D

Comment (robikovacs) [reply my-app/src/components/HelloWorld.vue]: Correct @florinionce !

File: my-app/public/index.html (hunk 4 above)
Comment (andiwolfpack) [reply my-app/public/index.html]: Vue magic 👍

**Comment by florinionce** [comment]
Do you know vuex? Pls for a project

**Comment by andiwolfpack** [comment]
Yeah sure guys I can help you out. I know plenty of Vue

**Pull request merged**